css/properties.schema.json

```diff
@@ -49,6 +49,7 @@
         "logicalHeightOfContainingBlock",
         "logicalWidthOfContainingBlock",
         "logicalHeightOrWidthOfContainingBlock",
+        "mapToRange0To1",
```

Comment:
When adding this new percentages enum representing `map to the range [0,1]`, I dropped 'the' when concatenating it and wrote the range in the same fashion as existing entries. See for example `specifiedValueNumberClipped0To1`

As noted in the PR body, it might be worth triple checking if this addition to the schema is correct.